src/api/controller/front.js

```diff
@@ -212,29 +217,13 @@ if (config.userAuth) {
 }
 
 app.use(handleRender);
-
-if (process.env.NODE_ENV === 'development') {
-    app.use(
-        koaWebpack({
-            config: webpackConfig,
-            dev: {
-                publicPath: '/',
-                stats: {
-                    colors: true,
-                },
-                quiet: false,
-                noInfo: true,
-            },
-            hot: {
-                log: global.console.log,
-                path: '/__webpack_hmr',
-                heartbeat: 10 * 1000,
-            },
-        }),
-    );
-} else {
-    app.use(mount('/', serve(path.resolve(__dirname, '../../build'))));
-    app.use(mount('/', serve(path.resolve(__dirname, '../../app/custom'))));
-}
+app.use(
+    route.get('/admin', async ctx => {
+        ctx.body = adminIndexHtml;
+    }),
+);
+
+app.use(mount('/', serve(path.resolve(__dirname, '../../build'))));
```

Comment:
Does this mount should be available on dev?

Reply:
Since it does not interfer, and could be useful to check the build, might as well leave it ?